keyboards/clueboard/66/rev2/config.h

```diff
@@ -1,7 +1,4 @@
-#ifndef CLUEBOARD_66_REV2_CONFIG_H
-#define CLUEBOARD_66_REV2_CONFIG_H
-
-#include "config_common.h"
+#pragma once
```

Comment:
Line 38 of this file needs to be deleted.

Reply:
1dd51c55f fixes this